### PR TITLE
Design and Information update

### PR DESCRIPTION
**Pull request by MrGKanev**
Done:
* Changed the homepage text and images
* Header on all needed pages
* Simplifying design where needed
* Added Motamo analytics
* Commented out try out
* Added pricing page

### CODE REVIEW

**Comment by MrGKanev** [comment]
I'm breaking the ticket a bit so it doesn't become a bigger mess. #84 will continue the meta changes after we think of a better solution.

File: resources/views/themes/bimbala/layouts/app.blade.php [hunk 1]
```diff
@@ -57,6 +57,40 @@
     @include('theme::partials.header')
 
     <main class="flex-grow overflow-x-hidden">
+        @unless (Request::path() == '/' ||  Request::path() == 'login'||  Request::path() == 'pricing' || Request::path() == 'register' ||Request::path() == 'dashboard' || Request::is('account/*')|| Request::is('password/*') || Request::is('@*') || Request::is('blog/*'))
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/layouts/app.blade.php:60]:
```suggestion
        @unless (Request::is(['/', 'login', 'pricing', 'register', 'dashboard', 'account/*', 'password/*', '@*', 'blog/*']))
```

File: resources/views/themes/bimbala/layouts/app.blade.php [hunk 2]
```diff
@@ -57,6 +57,40 @@
...
+            <div class="bg-gray-900">
+                <div class="max-w-7xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:px-8 lg:flex lg:justify-between">
+                    <div class="max-w-xl">
+                        <h2 class="text-4xl font-extrabold text-white sm:text-5xl sm:tracking-tight lg:text-6xl">{{ $seo['seo_title']}}</h2>
+                        <p class="mt-5 text-xl text-gray-300">{{ $seo['seo_description'] }}</p>
+                    </div>
+                    @if(Request::path() == 'blog' || Request::path() == 'use-case' || Request::path() == 'about-us')
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/layouts/app.blade.php:67]:
```suggestion
                    @if(Request::is(['blog', 'use-case', 'about-us']))
```

File: resources/views/themes/bimbala/layouts/app.blade.php [hunk 3]
```diff
@@ -57,6 +57,40 @@
...
+                    <div class="mt-10 w-full max-w-xs">
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/layouts/app.blade.php:68]: This looks good on mobile, but not on desktop:
image

File: resources/views/themes/bimbala/layouts/app.blade.php [hunk 4]
```diff
@@ -57,6 +57,40 @@
...
+                        <label for="currency" class="block text-base font-medium text-gray-300">Subscribe to our e-mails</label>
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/layouts/app.blade.php:69]:
```suggestion
                        <p class="block text-base font-medium text-gray-300">Subscribe to our e-mails</p>
```

File: resources/views/themes/bimbala/partials/footer.blade.php [hunk 5]
```diff
@@ -166,7 +166,7 @@ class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:
                     <a href="/privacy-policy" class="pr-5 mr-5 text-gray-500 no-underline bg-transparent border-r border-gray-700 cursor-pointer">Privacy Policy</a>
                 </li>
                 <li class="box-border block font-semibold text-left">
-                    <a href="/disclaimers" class="pr-5 mr-5 text-gray-500 no-underline bg-transparent border-r border-gray-700 cursor-pointer">Disclaimers</a>
+                    <a href="/gdpr" class="pr-5 mr-5 text-gray-500 no-underline bg-transparent border-r border-gray-700 cursor-pointer">GDPR</a>
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/partials/footer.blade.php:169]: The page is missing.

File: resources/views/themes/bimbala/about_us.blade.php [hunk 6]
```diff
@@ -3,47 +3,48 @@
...
             <div class="relative col-span-1 w-full h-full flex justify-center items-center">
-                <div class="h-full w-1 bg-white"></div>
-                <div class="absolute w-8 h-8 rounded-full bg-white z-10 text-black text-center"></div>
+                <div class="h-full w-1 bg-black"></div>
+                <div class="absolute w-8 h-8 rounded-full bg-black z-10 text-black text-center"></div>
             </div>
             <div class="col-span-4 w-full h-full"></div>--}}
 
 
                 <!-- Stack 2 -->
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/about_us.blade.php:41]: Let's darken this stack, taking into account the new design.

File: resources/views/themes/bimbala/about_us.blade.php [hunk 7]
```diff
@@ -53,17 +54,17 @@
                 </div>
 
                 <!-- Stack 1 -->
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/about_us.blade.php:56]: Let's darken this stack, taking into account the new design.

**Review by l-alexandrov — CHANGES_REQUESTED** [review]

File: resources/views/themes/bimbala/partials/footer.blade.php (hunk 5 above)
Comment (MrGKanev) [reply resources/views/themes/bimbala/partials/footer.blade.php:169]: It is a Voyager one.

File: resources/views/themes/bimbala/layouts/app.blade.php (hunk 3 above)
Comment (MrGKanev) [reply resources/views/themes/bimbala/layouts/app.blade.php:68]: mhm, have you tried rebuilding the css? - for me it looks fine on both (mobile and desktop)

File: resources/views/themes/bimbala/about_us.blade.php (hunk 6 above)
Comment (MrGKanev) [reply resources/views/themes/bimbala/about_us.blade.php:41]: I'm not sure I understand your suggestion. I wasn't really happy with how it turned out so If you have an idea of making it better I'm open to suggestions.

Comment (l-alexandrov) [reply resources/views/themes/bimbala/about_us.blade.php:41]: My intention is adding more contrast here, so I think something like that would be fine:
image
It only lacks of any images. If we find one, would be great.

File: resources/views/themes/bimbala/layouts/app.blade.php (hunk 3 above)
Comment (l-alexandrov) [reply resources/views/themes/bimbala/layouts/app.blade.php:68]: Yes, after rebuilding the css works fine. Let's upload the changed views along with the rebuilt assets.

File: resources/views/themes/bimbala/about_us.blade.php (hunk 6 above)
Comment (MrGKanev) [reply resources/views/themes/bimbala/about_us.blade.php:41]: Just push it as you want. We can make a ticket for future reworking.

**Review by l-alexandrov — APPROVED** [review]

**Pull request merged**